### ISSUE TITLE
1/8 EES brochure for Comparator Condition Session(s)

### ISSUE THREAD

**Issue by branscombj** [lzim/teampsd#811]
**1. Add description:** 
We need to accredit the comparator condition intervention (data UI only - in either 2 or 12 sessions depending on study). I created a straw man for this here: https://github.com/lzim/teampsd/blob/Comparitor-Intervention/ees_workgroup/master_brochures/data_session_ees_brochure.md

I started from the MTL S02 brochure, but unfortunately I renamed the file instead of keeping it as a pull request with the changes I made visible. Grr. I will work in that same branch on modifying the See guides for that.

If you want to add this to our Document Tracker, I would suggest adding under each of the existing columns a new card for, maybe data_s01 and data_s02.. ?

<< Paste screenshots here>>
 
**2. Click on the right:**
a. **Projects** - assign to **issue_tracker.**
b. **Assignee** - assign relevant team members.
c. **Labels**- review and select all potential _MTL_ dependencies.
d. **Milestones** - select the dependent milestone.

-----------------------------------------------------------------------------------------
**Workgroups leads**
**Identify Constraints (check all that apply):**
- [ ]   Capacity
- [ ]   Cost
- [ ]   Schedule
- [ ]   Interference
- [ ]   Capability
- [ ]   Other (describe)

_Add estimated person-hours to complete:_

**Edit the due date (if necessary)**

**Comment by branscombj** [comment]
no comment just adding tags etc

**Comment by lzim** [comment]
Hi @branscombj 

@staceypark are divvying up tasks for the new Red and Blue mapping of mtl.how on #830 

I wanted to ask you about Red Comparator Guides before the holidays. 
**_Did you already start to draft a session 2 (aka "Red Part I") and session 3 (aka "Red Part II")?_**

We've decided to have 
1. **Red Part I** (aka session 2) and
2. **Red Part II** (aka session 3) 
We'll do part 1 and 2 in two sessions for R01 Red. 
We'll use the part 1 guide that we visit 6 times for 6 sessions and a part 2 guide that we visit 6 times for 6 sessions in IIR Red.

The branch showing this part 1 and part 2 approach that will soon become a pull request to the mtl Master here: https://github.com/lzim/mtl/tree/lzim-patch-2/red

**Comment by branscombj** [comment]
**Some dependencies arising from Red-Blue tracks:**
- [ ] If the button name in the Data UI, "Create Team Data Table for Sim UI" gets changed to "Create Team Data Table", replace the gif(s) that show the old name, e.g. in Session 02 Step 5.
- [ ] Also clean up verbiage anywhere that names it in the scripts and see guides.

**For MTL Red resources:**
- [ ] Remove past-future graphic

**Comment by lzim** [comment]
Thanks @branscombj 

> If the button name in the Data UI, "Create Team Data Table for Sim UI" gets changed to "Create Team Data Table", replace the gif(s) that show the old name, e.g. in Session 02 Step 5.
- @staceypark already made the button change. Stacey, are you able to create .gifs?

- @branscombj for the guide verbiage and graphic removal, do you have any time to take a first stab at a pull request for that in in our facilitator folder?

**Comment by staceypark** [comment]
Great catch @branscombj. I worked with Andrew to create a new Data UI with the updated button name. 
- [ ] I can work on creating the updated gifs. 

You've already created a new Master Brochure & Learning Objectives for Red Part I (revised Session 2)  (I had edited to have deletions are bolded, and additions italicized for Elizabeth's benefit - but based on conversation yesterday maybe we don't need that). Could you also create one for Red Part II (revised Session 3)?

> **Some dependencies arising from Red-Blue tracks:**
> 
> * [ ]  If the button name in the Data UI, "Create Team Data Table for Sim UI" gets changed to "Create Team Data Table", replace the gif(s) that show the old name, e.g. in Session 02 Step 5.
> * [ ]  Also clean up verbiage anywhere that names it in the scripts and see guides.
> 
> **For MTL Red resources:**
> 
> * [ ]  Remove past-future graphic

**Comment by branscombj** [comment]
> Hi @branscombj
> 
> @staceypark are divvying up tasks for the new Red and Blue mapping of mtl.how on #830
> 
> I wanted to ask you about Red Comparator Guides before the holidays.
> **_Did you already start to draft a session 2 (aka "Red Part I") and session 3 (aka "Red Part II")?_**
> 
> We've decided to have
> 
> 1. **Red Part I** (aka session 2) and
> 2. **Red Part II** (aka session 3)
>    We'll do part 1 and 2 in two sessions for R01 Red.
>    We'll use the part 1 guide that we visit 6 times for 6 sessions and a part 2 guide that we visit 6 times for 6 sessions in IIR Red.
> 
> The branch showing this part 1 and part 2 approach that will soon become a pull request to the mtl Master here: https://github.com/lzim/mtl/tree/lzim-patch-2/red

@staceypark 
[MTL Red.docx](https://github.com/lzim/teampsd/files/3984441/MTL.Red.docx)

@lzim @dlkibbe @dlounsbu  - Could we consider having just one MTL Red "session" with a generic set of learning objectives, agenda, and post-test questions - thus one brochure, one see guide, and one say guide. This would simplify our interface with EES for accreditation and provide maximum flexibility to facilitators. I have produced a draft See Guide, with specific title, "Today we are *Modeling to Learn*..." line, Done/Dos, learning objectives, in-session exercise, and Done/Do.  Apologies that it's in Word.
We would not necessarily cover all of the steps in the first session with a team. In the 2-session version of MTL Red, we would do the first 6 steps in the first session and the next 6 the next session. In the 12-session version we could spend as much time as appropriate wherever needed, moving back and forth across splash page, team data, and team params, and possibly pulling a new dataset after a few months.

**Comment by lzim** [comment]
@branscombj and @staceypark 

I worry that the tradeoff on ease of accreditation may not be worth it in terms of measuring fidelity for each session's content for the research study.

In other words, if we only cover half the material, it will by definition systematically drive down the fidelity ratings for the comparator arms. We proposed in the R01 that it would be session 2 and session 3.

I thought it was clearer for both facilitator and learner if we divvy up the tasks of... 
part 1. ...carefully building your team and reviewing data, viz and count tabs, and separate it from the... 
part 2. ...review of the CC, MM, PSY, AGG and SP Params team data tables.

**Comment by branscombj** [comment]
OK, I hear you @lzim . So, @dlkibbe we need 10 post-test questions that will be posed to learners multiple times around
> part 1. ...carefully building your team and reviewing data, viz and count tabs

and 10 around
> part 2. ...review of the CC, MM, PSY, AGG and SP Params team data tables.

I will call the sessions/brochures MTL Red Part 1 and MTL Red Part 2 and will hone existing learning objectives to do this. In answer to 

>Did you already start to draft a session 2 (aka "Red Part I") and session 3 (aka "Red Part II")?

Yes, I will put my edits in as a pull request in the branch listed above.

**Comment by branscombj** [comment]
Despite its name and mynotes on the initial pull request #855 , the branch branscombj_red_part1 contains new ees brochures and learner see guides for both Part 1 and Part 2.
@lzim @dlkibbe @staceypark @ritahitching please review and see if these work for Red, and what I may have missed in converting them from Blue. Thanks!

**Comment by branscombj** [comment]
I'm not sure how to keep this straight on Github, but the real MTL RED POST-TESTS for review are here: https://github.com/lzim/teampsd/tree/kibbed-red-part2

The MTL RED BROCHURES and SEE GUIDES are here: https://github.com/lzim/teampsd/tree/branscombj-red-part1

**Comment by staceypark** [comment]
@branscombj @dlkibbe I've merged the pull request for the Red SEE, Red SAY, Red Post Test, and Red Brochures. As such, I am closing this.

**Comment by staceypark** [comment]
@branscombj Have we already sent the Comparator arm brochures to EES? I'm noticing the learning objectives are more different than they need to be

**Comment by branscombj** [comment]
@staceypark We have, but I don't think it's a problem to replace them, just let me know. I did make some edits to the See/Say guides for Red 1-2 vs Blue 02 and 03, including learning objectives, to account for the fact that they won't be doing S01 or S04 or any simulation, and for the fact that some who get Red will do each session 6 times. But it's fine if you think they could be more similar.

**Comment by staceypark** [comment]
We do not need to change the learning objectives. Can close this again.

**Comment by lzim** [comment]
@lijenn Do you need anything from me to send the _MTL_ Red brochures to Joe before we meet with him again?
Please cc @branscombj and I on any emails.

**Comment by branscombj** [comment]
> @lijenn Do you need anything from me to send the _MTL_ Red brochures to Joe before we meet with him again?
> Please cc @branscombj and I on any emails.

We will need to send both the *brochures* and the *post-tests* for MTL Red Part I and Part II, and indicate the correct responses on the post-test. Typically, we signify the correct answer to EES by showing it as "a" for each question, and just ask them to randomize the response options for administration.

I am looking at what's in the TeamPSD>EES repo to see if they look like the finalized versions discussed above. I think the LOs for Part II need to be revised to align with MTL Blue S03; now checking on the post-tests.

**Comment by branscombj** [comment]
@lijenn This spreadsheet may help for our communications with EES (?) - sheets 2 & 3 also are set up for inserting the links to resources for teams - though you may have another mechanism set up for this, perhaps more automated through github.
[EESandTeamLinksTables.xlsx](https://github.com/lzim/teampsd/files/5822186/EESandTeamLinksTables.xlsx)

**Comment by lijenn** [comment]
@branscombj Hope you had a great long weekend, Jane!

Thanks for catching that the brochures and SEE/SAY guides needs edits, would you be able to help edit those? 
If not, we can try to divvy it up.
How did the post-tests look, if you got the chance to review them?

The excel sheet is perfect actually! I can't think of an automated way of managing this yet...but that'll be a question that I'll ask our CRM Dynamics support team.

**Comment by branscombj** [comment]
> Thanks for catching that the brochures and SEE/SAY guides needs edits, would you be able to help edit those?
>
Hi @lijenn - I can edit, but can you help me with the process? These have current working versions living at mtl.how and copies in teampsd; also I'm not sure how changes tie in to the linter?

**Comment by lijenn** [comment]
@branscombj Hi Jane, James is still working/tweaking the linter/branching/review process due to some bugs. Let's make the edits by just creating your own branch and making the pull request once you've made the edits, like how we did it before. Once the linters/branching process is updated, we'll be sure to get it in the Manual so everyone is oriented to it!

The **brochures and post-tests** should only be here in the **teampsd repo** in the ees_workgroup folder. 
And we have moved to just editing/updating the **SEE/SAY guides** all on the **mtl.how repo**, so the ones in the teampsd repo are not in use anymore.

**Comment by branscombj** [comment]
Hi @lijenn - I have updated the LOs in MTL Red Part I and Part II to be exactly the same as those for MTL Blue S02 and S03 respectively, in both the SEE and SAY guides at mtl.how. With the additional tweak to the word order of the 2nd LO in each of those two to make it read easier. That edit I carried back into MTL Blue S02 and S03 SEE and SAY guides - all in the https://github.com/lzim/mtl/tree/red-learningobjectives-update branch.

**Comment by lijenn** [comment]
@branscombj Thank you Jane! I'll give it a review and push the commits before the end of the day. And happy inauguration day :)

**Comment by branscombj** [comment]
> And happy inauguration day :)

INDEED!!! WHEW! We went outside whooping and hollering once the oath was said and my husband blew a fanfare on his trumpet!

**Comment by branscombj** [comment]
> @branscombj Thank you Jane! I'll give it a review and push the commits before the end of the day. And happy inauguration day :)

Also made those updates to the ees brochures in the ees_for_red branch in TeamPSD repo.

**Comment by branscombj** [comment]
**LOGO links question**
Need to check the links to the logos in the post-tests IF those are going to be pulled from EES and intend to have the MTL logo on them. What I saw was that the MTL Blue S03 post-test has code for the logo at the top that is broken.

**Comment by lijenn** [comment]
Good question Jane, I think Joe might know the answer to that when we meet (tentatively next Mon., still waiting on his response fyi).

**Comment by branscombj** [comment]
Hi @lijenn Here are the MTL Red Part I and II post tests. 
[MTL Red PostTest.docx](https://github.com/lzim/teampsd/files/5870128/MTL.Red.PostTest.docx)
I was going to put them [here](https://github.com/lzim/teampsd/tree/ees_for_red/ees_workgroup/post_tests) as a pull request, but that was going to take me a while to figure out - sorry! There are 3 Red post-test files in there now and those should all go and be replaced by these. I think Joe et al may want them in two Word files (part I and part II). I know they'll want to be tipped off that the correct answer is **a** in each case, as they are presented here; and that we would like them to randomize the response options.

**Comment by lijenn** [comment]
Got it about the post tests in separate docs and that we'll need them to randomize the response options, thanks @branscombj!
